src/controller/product.controller.ts

```diff
@@ -267,6 +267,48 @@ export default class ProductController extends BaseController {
     return this.success(res, '--product_delete/success', 'Product has been deleted successfully', 200);
   }
 
+  async createCategory(req: Request, res: Response) {
+    const userId = (req as any).user['id'];
+    console.log(userId);
+    // const userId = "d7955c27-4d61-4cd6-a6bb-e6402151d51f"
+    const { error, value } = createCategorySchema.validate(req.body);
+
+    if (error) {
+      return res.status(400).json({ error: error.message });
```

Comment:
this section:

```js
 if (error) {
      return res.status(400).json({ error: error.message });
    }
```

use `this.error` for error response and `this.success` for succes response

Reply:
fixed